packages/dx-react-grid/docs/reference/table-filter-row.md

```diff
@@ -32,6 +32,7 @@ Field | Type | Description
 filter | [Filter](filtering-state.md#filter) | Filtering options that are applied to a column.
 onFilter | (filter: [Filter](filtering-state.md#filter)) => void | An event that initiates applying a new filter to a column.
 column | [Column](grid.md#column) | A column.
+columnFilteringEnabled | boolean | Specifies whether filtering by a column is enabled.
```

Comment:
The name is too verbose. Maybe transform it to `disabled`?

Reply:
`disable` isn't clear in this case. Cell is disabled. What does it mean?

Reply:
Is the `filteringDisabled` name completely describes the meaning?

Reply:
Why should we invert `enabled` to `disabled`? Let's keep just `filteringEnabled`.